roboteam_ai/src/utilities/Constants.cpp

```diff
@@ -0,0 +1,187 @@
+//
+// Created by mrlukasbos on 7-2-19.
+//
+
+#include "Constants.h"
+
+namespace rtt {
+namespace ai {
+
+// static variable initialization
+bool Constants::useGrSim = false;
+std::map<std::string, double> Constants::doubles;
+std::map<std::string, bool> Constants::bools;
+std::map<std::string, int> Constants::integers;
+std::map<std::string, QColor> Constants::colors;
+
+double Constants::getDouble(const std::string &name) {
+    if (doubles.find(name) != doubles.end() ) {
+        return doubles[name];
+    }
+    std::cerr << "Double constant not found: " << name.c_str() << std::endl;
+    return 0.0;
+}
+
+int Constants::getInt(const std::string &name) {
+    if (integers.find(name) != integers.end() ) {
+        return integers[name];
+    }
+    std::cerr << "Integer constant not found: " << name.c_str() << std::endl;
+    return 0;
```

Comment:
Do you want these to return a default? This could definitely break stuff and you might not always check the error log.

Reply:
well... i have to